### DISCUSSION TITLE
Bucket CORS configuration

### DISCUSSION THREAD

**Discussion by tegarmonster** [minio/minio#20555] [Q&A] [answered]
Hi, 
How to added bucket CORS configuration using mc?

thanks

**Comment by harshavardhana** [comment]
For bucket cors feature you need to use our EOS release.

**Reply by tegarmonster** [reply]
Hi @harshavardhana ,
sorry what is that? is the minio latest version?

**Reply by marktheunissen** [reply, marked as answer]
@tegarmonster it's the paid plan: https://min.io/product/enterpriseoverview

**Reply by tegarmonster** [reply]
hi @marktheunissen ohh okay, thanks!
